pennylane/transforms/mitigate.py

```diff
@@ -22,6 +22,12 @@
 from pennylane.tape import QuantumScript, QuantumTape
 from pennylane.transforms import transform
 
+has_jax = True
+try:
+    import jax.numpy as jnp
+except ImportError:
+    has_jax = False
+
```

Comment:
@rmoyard Do I understand it correctly from your review that once `qml.math` is used, this is no longer needed?

Reply:
Removed in https://github.com/PennyLaneAI/pennylane/pull/5972/commits/b60ffc442cc71c1bc6152152edd4638d12dde9e3.